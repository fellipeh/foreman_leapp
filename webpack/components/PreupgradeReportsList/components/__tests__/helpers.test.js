import { testSelectorsSnapshotWithFixtures } from '@theforeman/test';

import { presentSeverity } from '../helpers';

const fixtures = {
  'should return tag info': () => presentSeverity('low'),
};

describe('PreupgradeReportList/helpers', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
