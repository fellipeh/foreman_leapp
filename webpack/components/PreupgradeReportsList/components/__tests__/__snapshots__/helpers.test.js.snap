// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreupgradeReportList/helpers should return tag info 1`] = `"#39a5dc"`;
